mvc/overview.rst

```diff
@@ -1,8 +1,68 @@
-.. include:: /../common/stub-topic.txt
+Overview of ASP.NET MVC
+=======================
+By `Tom Archer`
 
-|stub-icon| Overview of ASP.NET MVC
-===================================
+Based on the `MVC architectural pattern <https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller>`_, the ASP.NET MVC framework is a lightweight, cross-platform, highly testable application-development framework that separates an application or Web API into three main components: model, view, and controller:
 
-.. include:: /../common/stub-notice.txt
+- :doc:`Models </models/index>` represent data that the user is working with in your application. For example, in an inventory application, one of the model objects might be a Product object that is tasked with retrieving information from a database, operating on that information, and then writing the updated information back to the database. In small applications, the model is often a conceptual separation instead of a physical one. For example, if the application only reads a dataset and sends it to the view, the application does not have a physical model layer and associated classes. Instead, the controller might work directly with the dataset, which takes on the role of the model.
 
-.. _issue: https://github.com/aspnet/Docs/issues/113
+- :doc:`Views </views/index>` are responsible for displaying the application's user interface (UI). Typically, this UI is created from the model data. An example would be an edit view of a Products table that displays text boxes, drop-down lists, and check boxes based on the current state of the Product object being represented in the view.
+
+- :doc:`Controllers </controllers/index>` process incoming requests, work with the model, and ultimately select a view that displays the user interface. In an MVC application, the view only displays information; the controller handles and responds to user input and interaction. For example, the controller handles query-string values, and passes these values to the model, which in turn might use these values to query the database.
```

Comment:
Need to distinguish client-side and server-side interactions here - MVC doesn't handle anything that happens on the client side (e.g. through JS), only the HTTP request.

Not sure what "handles query-string values" means in this context.